shared/actions/chat2/index.js

```diff
@@ -1742,7 +1733,12 @@ const markThreadAsRead = (
 const deleteMessageHistory = (action: Chat2Gen.MessageDeletePayload, state: TypedState) => {
```

Comment:
we should update the rpc so we don't have to pass a message id anymore and bookkeep this. The UX is just that we want the entire thing cleared so we should be able to just call some rpc that does that and takes just the conversationIDKey

Reply:
I considered that, but won't this operation then be broken for older clients? e.g., admin has this version and sends clear message with no message ID, and another team member has an older client. If this case isn't a big deal then yeah I agree.

Reply:
i'm talking about the rpc between the app and the daemon, which is just local so it doesn't affect other clients. the daemon can derive what the last message is